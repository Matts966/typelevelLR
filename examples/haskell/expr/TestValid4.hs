
import Expr

main :: IO ()
main = print $ begin |> lp |> num 1 |> add |> num 2 |> rp |> mul |> num 3 |> end
