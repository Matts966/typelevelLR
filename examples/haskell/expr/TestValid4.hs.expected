TToE (Mul (FToT (Paren (Add (TToE (FToT (Num 1))) (FToT (Num 2))))) (Num 3))
